devtools/gradle/src/main/java/io/quarkus/gradle/QuarkusPluginExtension.java

```diff
@@ -98,7 +98,7 @@ public void setLibDir(String libDir) {
 
     public String finalName() {
         if (finalName == null || finalName.length() == 0)
-            return project.getName();
+            return project.getName() + "-" + project.getVersion();
```

Comment:
Is there any specific reason this was changed?

Reply:
Yeah, see the full commit message of this commit.

Reply:
Duh, I should have checked that :)